server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AbstractEndpoint.java

```diff
@@ -344,12 +341,19 @@ public void before(final Request the_request, final Response the_response) {
   } 
   
   /**
-   * The after filter for this endpoint. By default, it attempts to commit 
+   * The after filter for this endpoint. Currently, the implementation is empty.
+   */
+  public void after(final Request the_request, final Response the_response) {
+    // skip
+  }
+  
+  /**
+   * The afterAfter filter for this endpoint. By default, it attempts to commit 
    * any open transaction (this makes writing endpoint code more straightforward,
    * as the vast majority of endpoints will never have to deal with transactions
    * themselves).
    */
-  public void after(final Request the_request, final Response the_response) {
+  public void afterAfter(final Request the_request, final Response the_response) {
     // try to take the transition for this endpoint in the ASM and save it to the DB
     // note that we do not try to commit when we have an error code in the response
     if (the_response.status() == HttpStatus.OK_200 && 
```

Comment:
Not necessarily for this PR, but this should probably check that the status is 2xx, not 200. For example, a 201 would be appropriate for the success path of many of our endpoints.